src/AdoNet/Orleans.Reminders.AdoNet/SiloHostBuilderReminderExtensions.cs

```diff
@@ -59,30 +61,12 @@ public static ISiloHostBuilder UseAdoNetReminderService(this ISiloHostBuilder bu
         /// <returns>
         /// The provided <see cref="ISiloHostBuilder"/>, for chaining.
         /// </returns>
-        public static IServiceCollection UseAdoNetReminderService(this IServiceCollection services, Action<AdoNetReminderTableOptions> configure)
+        public static void UseAdoNetReminderService(this IServiceCollection services, Action<OptionsBuilder<AdoNetReminderTableOptions>> configureOptions)
```

Comment:
return type should be IServiceCollection, no?

Reply:
no need to change public interface now

Reply:
The public interface needs to change anyway, since it was wrong. The overload which takes a connection string simply cannot work without a second call to also configure the Invariant.

The whole method should be removed

Reply:
? didn't you already removed that wrong method, which is just below this one. This one is fine , right? I was just wondering why you changed the return type.